war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -891,14 +877,6 @@ function rowvgStartEachRow(recursive,f) {
         makeButton(e, expandButton);
     });
 
-    // scripting for having default value in the input field
-    Behaviour.specify("INPUT.has-default-text", "input-has-default-text", ++p, function(e) {
```

Comment:
Same as above

Reply:
not used https://github.com/search?q=org%3Ajenkinsci+has-default-text&type=Code